packages/xrpl/test/integration/requests/fee.test.ts

```diff
@@ -26,17 +28,21 @@ describe('fee', function () {
       const request: FeeRequest = {
         command: 'fee',
       }
+      const referenceFee = xrpToDrops(await getFeeXrp(testContext.client, 1))
+      const MEDIAN_FEE_MULTIPLIER = 500
       const response = await testContext.client.request(request)
       const expected = {
         id: 0,
         result: {
           current_ledger_size: '0',
           current_queue_size: '0',
           drops: {
-            base_fee: '10',
-            median_fee: '5000',
-            minimum_fee: '10',
-            open_ledger_fee: '10',
+            base_fee: referenceFee,
+            median_fee: new BigNumber(referenceFee)
```

Comment:
We can omit the `median_fee` field from the deep-equality comparison. Quoting from the docs: `An approximation of the median transaction cost among transactions included in the previous validated ledger, represented in drops of XRP.`

Since this value depends on the transactions-mix of the previous validated ledger, it is not informative for integration test status. Docs: https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/server-info-methods/fee#response-format